README.md

```diff
@@ -272,6 +273,12 @@ Alternatively you can pull from [Docker Hub](https://hub.docker.com/r/kuchy/pack
 docker pull kuchy/packtpub-crawler
 ```
 
+### cronjob
+Add this to your crontab to run the job daily at 9:
+```
+00 09 * * * /usr/bin/python PATH_TO_CRAWLER/script/spider.py --config PATH_TO_CRAWLER/config/prod.cfg --all --extras --notify join >> /tmp/packtpub.log 2>&1
+```
+
```

Comment:
Ah, I forgot to mention it in the comment, sorry. Can you run the command without cron? What about the log in tmp?